uber/models.py

```diff
@@ -1160,8 +1160,8 @@ def age_group_conf(self):
             for val, age_group in c.AGE_GROUP_CONFIGS.items():
                 if val != c.AGE_UNKNOWN and age_group['min_age'] <= attendee_age <= age_group['max_age']:
                     return age_group
```

Comment:
But making this an ``else`` statement, we weren't executing it if we entered the previous if-block but didn't belong to any configured age group.